app/assets/translations/appConstants/bebbo/en.ts

```diff
@@ -351,18 +351,18 @@ const en = {
                 "settingExportError": "Error while backing up data. Please try again.",
                 "settingExportSuccess": "Data backed up successfully",
                 "alertForModifyMeasures": "Attention",
-                "alertForModifyMeasuresTitle": "Selecting this date will modify existing measures",
+                "alertForModifyMeasuresTitle": "Selecting this date will modify existing measures.",
                 "alertForModifyMeasuresOk": "OK",
                 "SyncOnLoadPopupTitle": "Update app content",
                 "SyncOnLoadPopupText": "Do you want to update app content now?",
                 "SyncOnLoadCancelPopUpBtn": "Cancel",
                 "SyncOnLoadRetryBtn": "Update",
-                "cdNoti1": "{{childName}} has entered period of {{periodName}} - track your child's development",
-                "cdNoti2": "Check what milestones {{childName}} has reached in the period of {{periodName}}",
-                "gwNoti1": "{{childName}} is growing - update growth measures",
-                "hcNoti1": "In this period, {{periodName}}, {{childName}} should have a health check up. Schedule a visit to the doctor",
-                "vcNoti1": "In the period, {{periodName}}, {{childName}} should receive vaccine:  ",
-                "alertForExistingMeasuresTitle": "Measurement already available for this date, please select another date",
+                "cdNoti1": "{{childName}} has entered period of {{periodName}} - track your child's development.",
+                "cdNoti2": "Check what milestones {{childName}} has reached in the period of {{periodName}}.",
+                "gwNoti1": "{{childName}} is growing - update growth measures.",
+                "hcNoti1": "In this period, {{periodName}}, {{childName}} should have a health check up. Schedule a visit to the doctor.",
+                "vcNoti1": "In the period, {{periodName}}, {{childName}} should receive vaccine:  ",
+                "alertForExistingMeasuresTitle": "Measurement already available for this date, please select another date.",
```

Comment:
`A measurement is already available for this date. Please select another date.` looks more complete.